scripts/init.sh

```diff
@@ -2,7 +2,17 @@
 
 set -e
 
+echo "Setup git hooks"
+git config --local core.hooksPath hooks/
+
+echo "Update submodules"
 git submodule update --init --recursive || true
+
+echo "Installing dev tools"
+go install github.com/ferranbt/fastssz/sszgen@v0.1.3
+
+echo "Install node packages"
 (cd web && pnpm install)
-go install github.com/ferranbt/fastssz/sszgen@latest
-ln -sf pre-commit.sh .git/hooks/pre-commit
+
+echo "Initialize foundry libraries"
+(cd contracts && forge install)
```

Comment:
Also explicitly calling `forge install` is not necessary, since the `git submodule update` command above initialises the submodules already